src/frontend/src/views/Upload/components/Metadata/components/Table/index.tsx

```diff
@@ -110,6 +113,16 @@ export default function Table({
     );
   }
 
+  const shouldShowGISAIDFields = reduce(
```

Comment:
I'm thinking maybe the following would do? 

```ts
shouldShowGISAIDFields = metadata.some((data) => Boolean(data?.submittedToGisaid))
```

This way we can early return `true` when we find the first element that has `submittedToGisaid === true` without going through the whole array like `reduce()` does. Secondly we avoid importing `reduce` and uses the native `Array.some()` method 😄 

[source](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some)